webroot/results/misc/evolution/update7205.php

```diff
@@ -33,6 +33,18 @@
 echo "<p></p>";
 
 #--- Define what to show in what order and with nice short names
+$events = dbQueryHandle("
+      SELECT id
+      FROM EVENTS
+      WHERE rank < 900
+      ORDER BY rank
+    ");
+
+$list = "";
+while($row = mysql_fetch_row($events)) {
+  $list .= $row[0] . ":cubing-icon event-" . $row[0] . " ";
+}
+#$whats = explode( ' ', $list);
 $whats = explode( ' ', '333:3x3 444:4x4 555:5x5 222:2x2 333bf:bld 333oh:hand 333fm:moves:1 333ft:feet minx:mega pyram:pyra sq1:sq1 clock:clock 666:6x6 777:7x7 magic:magic mmagic:master 444bf:4bld 555bf:5bld 333mbf:multi:1' );
```

Comment:
Shouldn't this be the commented line, and the previous one the uncommented line?